.github/workflows/ci.yml

```
@@ -39,7 +39,7 @@ jobs:
     - run:  npm ci
 
     - name: Tests
-      run:  npm run test -- --bundle=${{ matrix.bundle }}
+      run:  npm run test -- --no-lint --bundle=${{ matrix.bundle }}
```

Unless you think it's too obvious

```suggestion
      # run tests, but lint separately
      run:  npm run test -- --no-lint --bundle=${{ matrix.bundle }}
```